server/server.go

```diff
@@ -230,6 +257,12 @@ func (s *Server) ttrc(data interface{}) queue.SubQueue {
 // stop is called when the server is Stop()ped, cleaning up our additional
 // properties.
 func (s *Server) stop() {
+	s.sendSlackMessage("🟧 server stopped") //nolint:errcheck
```

Comment:
Might be worth pulling this string (and similar ones in `setdb.go` into a block consts so that they're all in once place.